Formula/fabio.rb

```diff
@@ -1,8 +1,8 @@
 class Fabio < Formula
   desc "Zero-conf load balancing HTTP(S) router"
   homepage "https://github.com/fabiolb/fabio"
-  url "https://github.com/fabiolb/fabio/archive/v1.5.11.tar.gz"
-  sha256 "628e7382b766f5b03ef72209f38694a723fe5838d044764c9619c3cc84696b67"
+  url "https://github.com/fabiolb/fabio/archive/v1.5.12.tar.gz"
+  sha256 "e09bbda58c491b90b367ca306e23d8c96ef63f41a1d1103d155d290c9fcdf6f0"
```

Comment:
```suggestion
  sha256 "0da898594baff6e2ecb2709a974b4a9787bba6d8bf79016e180413e9761c8fec"
```

Reply:
I have a script which downloads the tarball and computes the checksum. Have to dig in and find out why this is happening. Thank you and sorry for the mess. I'll fix it.

Reply:
I have an idea where this comes from and added a fix. I'm also running the suggested brew tests now in the script so that I should at least catch it should it happen again.

Thanks for all your work. Homebrew is awesome!